From b1856a2a3c5c3dcaafea6f1ff10522b97daae7ce Mon Sep 17 00:00:00 2001
From: Igor Grinberg <igrinberg@habana.ai>
Date: Tue, 21 Jan 2025 17:58:51 +0200
Subject: [PATCH] [FSW-13274] palad_bloader: do not build module_i2c

module_i2c_slave.c is not used in paladium - do not build it.
If we build this module, it brings deps not relevant to palad_bloader
and FS1.

Signed-off-by: Igor Grinberg <igrinberg@habana.ai>
Change-Id: I136b87adbc992c47476f2c84dac50483b6e4bcb7
---

diff --git a/app/palad_bloader/Makefile b/app/palad_bloader/Makefile
index b3798f6..49bbf59 100644
--- a/app/palad_bloader/Makefile
+++ b/app/palad_bloader/Makefile
@@ -37,6 +37,7 @@
 PALAD_BLOADER_SRCS_C := $(shell ls *.c)
 PALAD_BLOADER_SRCS_C += $(shell ls $(PREBOOT_ROOT_DIR)/lib/common/*.c)
 PALAD_BLOADER_SRCS_C := $(filter-out $(PREBOOT_ROOT_DIR)/lib/common/pif_ts.c,$(PALAD_BLOADER_SRCS_C))
+PALAD_BLOADER_SRCS_C := $(filter-out $(PREBOOT_ROOT_DIR)/lib/common/module_i2c_slave.c,$(PALAD_BLOADER_SRCS_C))
 PALAD_BLOADER_SRCS_C += $(shell ls $(HL_DEV_ARCH_COMMON)/common.c)
 PALAD_BLOADER_SRCS_C += $(shell ls $(HL_DEV_ARCH)/common.c)
 PALAD_BLOADER_SRCS_C += $(shell ls $(HL_DEV_ARCH)/$(HL_TARGET).c)
